autopep8

mlautman opened the pull request:
Results of running 
`autopep8 --max-line-length 120 --in-place $1` on each `.py` file

brawner commented on rqt_gui/src/rqt_gui/ros_plugin_provider.py:
```diff
@@ -82,12 +82,14 @@ def load(self, plugin_id, plugin_context):
             return None
         except Exception as e:
-            qCritical('RosPluginProvider.load(%s) exception raised in __builtin__.__import__(%s, [%s]):\n%s' % (plugin_id, attributes['module_name'], attributes['class_from_class_type'], traceback.format_exc()))
+            qCritical('RosPluginProvider.load(%s) exception raised in __builtin__.__import__(%s, [%s]):\n%s' % (
```
break into more lines

mlautman replied:
I would prefer to keep changes limited to autopep8

brawner commented on rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py:
```diff
@@ -69,7 +69,8 @@ def _check_for_master(self):
         self._wait_for_master_thread = threading.Thread(target=self._wait_for_master)
         self._wait_for_master_thread.start()
-        self._wait_for_master_dialog = QMessageBox(QMessageBox.Question, self.tr('Waiting for ROS master'), self.tr("Could not find ROS master. Either start a 'roscore' or abort loading the plugin."), QMessageBox.Abort)
+        self._wait_for_master_dialog = QMessageBox(QMessageBox.Question, self.tr('Waiting for ROS master'), self.tr(
```
bad line break

brawner commented on rqt_py_common/src/rqt_py_common/layout_util.py:
```diff
@@ -66,9 +66,9 @@ def alternate_color(list_widgets, colors_alter=[Qt.white, Qt.gray]):
-                                                                   divisor,
-                                                                   i_widget,
-                                                                   colors_num))
+                divisor,
```
align with str.format

brawner commented on rqt_py_common/src/rqt_py_common/rosaction.py:
```diff
@@ -363,34 +374,39 @@ def rosactionmain(mode=MODE_ACTION):
             raise yaml.constructor.ConstructorError("while constructing a mapping",
                                                     node.start_mark,
-                    "found unhashable key", key_node.start_mark)
+                                                    "found unhashable key", key_node.start_mark)
```
break for fourth argument

mlautman replied:
Not actually required for flake8 or pep8

brawner commented on rqt_py_common/src/rqt_py_common/rosaction.py:
```diff
@@ -598,7 +616,8 @@ def rosaction_cmd_prototype(args):
         if '::' in message_type:
             if not options.silent:
-                parser.error("rosactionproto does not understand C++-style namespaces (i.e. '::').\nPlease refer to msg/srv types as 'package_name/Type'.")
+                parser.error(
```
break into more lines

brawner commented on rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:
```diff
@@ -72,12 +72,12 @@ def get_full_grn(model_index):
                 grn_full = upper_grn + child_grn
                 rospy.logdebug('grn_full={} upper_grn={} child_grn={}'.format(
-                                               grn_full, upper_grn, child_grn))
+                    grn_full, upper_grn, child_grn))
```
align with str.format

brawner commented on rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:
```diff
@@ -123,12 +123,12 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
         while True:  # Loop per child.
             grn_curr = grn_prev + RqtRosGraph.DELIM_GRN + str(
-                                                            model_index.data())
+                model_index.data())
```
break at +

brawner commented on rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:
```diff
@@ -123,12 +123,12 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
             if (not child_qmindex.isValid()):
                 rospy.logdebug('!! DEADEND i_child=#{} grn_curr={}'.format(
-                                                           i_child, grn_curr))
+                    i_child, grn_curr))
```
align with str.format

brawner commented on rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:
```diff
@@ -146,9 +146,9 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
             list_grn_children_all = list_grn_children_all + list_grn_children
             rospy.logdebug('111 lennodes={} list_grn_children={}'.format(
-                                len(list_grn_children_all), list_grn_children))
+                len(list_grn_children_all), list_grn_children))
```
align with str.format

brawner commented on rqt_py_common/src/rqt_py_common/rqt_ros_graph.py:
```diff
@@ -146,9 +146,9 @@ def get_lower_grn_dfs(model_index, grn_prev=''):
+                len(list_grn_children_all), list_grn_children))
             rospy.logdebug('122 list_grn_children_all={}'.format(
-                                                        list_grn_children_all))
+                list_grn_children_all))
```
align with str.format

brawner commented on rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py:
```diff
@@ -69,14 +69,14 @@ def load_parameters(config, caller_id):
             # #2468 unify clear params to prevent error
             for param in roslaunch.launch._unify_clear_params(
-                                                          config.clear_params):
+                    config.clear_params):
```
don't break line

brawner commented on rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py:
```diff
@@ -69,14 +69,14 @@ def load_parameters(config, caller_id):
                 if code != 1:
                     raise RLException("Failed to clear parameter {}: ".format(
-                                                                         msg))
+                        msg))
```
don't break line

brawner commented on rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py:
```diff
@@ -95,7 +95,7 @@ def load_parameters(config, caller_id):
                 if code != 1:
                     raise RLException("Failed to set parameter: " +
-                                                "%s" % (msg))
+                                      "%s" % (msg))
```
don't break line

brawner commented on rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py:
```diff
@@ -149,7 +149,7 @@ def list_files(package, subdir, file_extension='.launch'):
         return [genmsg.resource_name(package, t)
                 for t in RqtRoscommUtil._list_types(
-                                                 path, file_extension)]
+            path, file_extension)]
```
don't break line

brawner commented on rqt_py_common/src/rqt_py_common/topic_completer.py:
```diff
@@ -57,7 +57,8 @@ def update_topics(self):
             message_class = roslib.message.get_message_class(topic_type)
             if message_class is None:
-                qWarning('TopicCompleter.update_topics(): could not get message class for topic type "%s" on topic "%s"' % (topic_type, topic_path))
+                qWarning('TopicCompleter.update_topics(): could not get message class for topic type "%s" on topic "%s"' %
```
break into more lines

brawner commented on rqt_py_common/test/test_rqt_ros_graph.py:
```diff
@@ -63,7 +63,7 @@ def setUp(self):
 
         #node_list = [node1, node1_1, self._node1_1_1, node1_1_2, node1_2]
-        #self._model.appendRow(node_list)
+        # self._model.appendRow(node_list)
```
commented code

brawner commented on rqt_py_common/test/test_rqt_ros_graph.py:
```diff
@@ -78,9 +78,9 @@ def test_get_upper_grn(self):
-                                                  self._node1_1.index(),
-                                                  '')),
-                         self._len_lower_grn_node1_1)
+            self._node1_1.index(),
```
align with .get_lower_grn_dfs

Commit:
autopep8

mlautman commented:
@brawner, I fixed up the changes you requested and also changed the line length limit to 120 as per ROS style guild lines. If these changes look good I would really like to merge them and move on. We can always apply additional stylistic guidelines and changes later on

brawner commented:
@mlautman Yah, let's move forward with porting code

dirk-thomas reviewed (approved):
LGTM
